db/migrate/20231103060018_fix_invalid_topic_user_bookmarked_data.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class FixInvalidTopicUserBookmarkedData < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec(<<~SQL)
+      UPDATE topic_users
```

Comment:
This query limits the scope of the fix a lot, since it's only fixing the known incorrect ones. To try and correct all the topic_users.bookmarked records in the database based on the condition of any bookmarks for a topic would be quite slow.